src/shared/components/palette/fixed-list/write-fixed-list-field.component.ts

```diff
@@ -15,38 +11,10 @@ export class WriteFixedListFieldComponent extends AbstractFieldWriteComponent im
   fixedListControl: FormControl;
 
   ngOnInit() {
-    this.instantiate();
     let notEmpty = this.caseField.value !== null && this.caseField.value !== undefined;
-    this.fixedListControl = this.registerControl(new FormControl(notEmpty ? this.caseField.getValue() : null));
-
-  }
-
-  /**
-   * THIS METHOD WILL NEED TO BE REMOVED AS REFACTORING OF
-   * https://tools.hmcts.net/jira/browse/RDM-4704
-   *
-   */
-  private instantiate(){
-    if(this.caseField){
-      var cf = new CaseField();
-      cf.field_type = this.caseField.field_type;
-      cf.id = this.caseField.id;
-      cf.display_context_parameter = this.caseField.display_context_parameter;
-      cf.hidden = this.caseField.hidden;
-      cf.label = this.caseField.label;
-      cf.order = this.caseField.order;
-      cf.field_type = this.caseField.field_type;
-      cf.value = this.caseField.value;
-      cf.hint_text = this.caseField.hint_text;
-      cf.security_label = this.caseField.security_label;
-      cf.display_context = this.caseField.display_context;
-      cf.display_context_parameter = this.caseField.display_context_parameter;
-      cf.show_condition = this.caseField.show_condition;
-      cf.show_summary_change_option = this.caseField.show_summary_change_option;
-      cf.show_summary_content_option = this.caseField.show_summary_content_option;
-      cf.acls = this.caseField.acls;
-      cf.wizardProps = this.caseField.wizardProps;
-      this.caseField = cf;
+    this.fixedListControl = this.registerControl(new FormControl(notEmpty ? this.caseField.value : null));
+    if(this.fixedListControl.value && this.fixedListControl.value.value){
+      this.fixedListControl.setValue(this.fixedListControl.value.value.code);
```

Comment:
I don't understand why this is needed. The caseField.value that we call on the line above (15) returns already the `code` isn't it? 
```if (this.field_type && this.field_type.type === 'DynamicList') {
      return this._value && this._value.value ? this._value.value.code : this._value;
    } ```

Reply:
When we choose a dynamic list and press contiue, we sanitisedynamiclists and add that to the formcontrol, for validate and going to next page now if we move previous or go back to the page from change link of check your answers it does not get selected as the formvalue contains enriched data ie { value, list_items} instead of 'List1'